src/main/java/org/isf/menu/model/UserGroup.java

```diff
@@ -40,12 +40,15 @@
 public class UserGroup extends Auditable<String> {
 
 	@Id
-	@Column(name="UG_ID_A")
+	@Column(name = "UG_ID_A")
 	private String code;
-	
-	@Column(name="UG_DESC")
+
+	@Column(name = "UG_DESC")
 	private String desc;
-	
+
+	@Column(name = "UG_DELETED")
+	private boolean deleted = false;
```

Comment:
```suggestion
	private boolean deleted;
```
assigning the value `false` is redundant.